### ISSUE TITLE
JetAppointment

### ISSUE THREAD

**Issue by dislokacia** [CrocoBlock/suggestions#2421]
Could you add the possibility to add the admin columns to JetAppointment and JetBooking plugins, please?
https://prnt.sc/whgtsv
For example, to add the admin column from the metafield in the booking/appointment CPT's.
thanks in advance

**Comment by TODO4EVER** [comment]
Yeah, Ability to change the columns would be amazing. Like I am building a site for a client that doesn't care about the customer email in the appointment, Like display User Name instead of User-email would make so much more sense.

**Comment by molnarbalazs950817** [comment]
+1

**Comment by moxet** [comment]
+1
Is there any updates on this issue.

**Comment by awebguy99** [comment]
+1 this is definitely needed.

**Comment by awebguy99** [comment]
Come on Crocoblocks, this feature needs to be implemented. You have custom fields in Jet Appointments, we need the ability to display these fields in the backend>Jet Appointment appointment list. **It's not viable for our clients to have to individually edit each appointment to see these fields.**

**Comment by zavants** [comment]
1+, this option is indispensable, which makes the plugin useless for more advanced appointment systems. We had to look for some alternatives even though we bought a Jetengine license and wanted to use the whole ecosystem.

**Comment by Crocoblock** [comment]
Now our team is working on more priority tasks. However, in the future, we will not exclude the possibility of adding this functionality to the core of our plugin.

Thank you for being so understanding.
